### DISCUSSION TITLE
currentSchema should be honored when config grafana database with url

### DISCUSSION THREAD

**Discussion by zhaizhengqing** [grafana/grafana#54194] [General]
url =postgres://zhaizhengqing:123654@192.168.56.1:5432/zhaizhengqing?currentSchema=grafana

When using url to config database for grafana, the currentSchema option should be honored.

![image](https://user-images.githubusercontent.com/4179759/186564891-6e7775a7-44c8-4a96-a104-efb3a6e63aca.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
